### DISCUSSION TITLE
I request a refund

### DISCUSSION THREAD

**Discussion by gamaflex** [aclap-dev/video-downloadhelper#1061] [License & Purchase requests]
### What's your issue?

I bought it to download my lesson videos but it gives me a download failed error. My only purpose was to download the lesson videos but I couldn't download any of them. I bought it today and I want a refund. I added the error message I received.
![Adsız](https://github.com/user-attachments/assets/76bf3870-264e-443c-a288-019d18bb10dc)

### The 10 first digits (only 10) of your license number or email address

9233f3b6b0

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
